src/Forms/PluginsGet.cs

```diff
@@ -285,6 +286,13 @@ private void buttonDownload_Click(object sender, EventArgs e)
 
         private void wc_DownloadDataCompleted(object sender, DownloadDataCompletedEventArgs e)
         {
+            // dispose the webclient if it's not the one declared as field (this will be the webclient used to
+            // download multiple plugin
+            if (sender != _wc)

```

Comment:
this will ensure all the web-clients created when downloading multiple plugin will be disposed